cmd/connector/main.go

```diff
@@ -1,3 +1,17 @@
+// Copyright © 2022 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
```

Comment:
Ah, missing in the template.:)

Reply:
Not really, I don't think the license header should be part of the template, that's only relevant for our in house connectors.

What _is_ missing though are `.golangci.yml` and `.goreleaser.yml`, and the injecting of the version that you commented on in the other comment.